### PR TITLE
First two sections of portfolio website

### PR DESCRIPTION
**Pull request by learnwithalfred**
This is the mobile screen display of the first two sections of the portfolio website

Screenshot 2022-04-20 at 11 20 44

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alfred Portfolio Website</title>
+  </head>
+  <body>
+    <main class="main">
+      <header class="header">
+        <a class="header-logo" href="#">Alfred</a>
+        <a href="javascript:void(0);" class="header-humbugger-icon">
+          <i class="fa fa-bars"></i>
+        </a>
```
Comment (prolajumokeoni) [inline index.html:26]: - [ ] Kindly wrap harmburger menu in nav tag and not a tag as this is part of the requirement 👍🏿

File: index.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+      </header>
+      <section class="developer">
+        <div class="developer-info">
+          <img
+            class="developer-info-shadow"
+            src="./images/header@2x.png"
+            alt=""
+          />
```
Comment (prolajumokeoni) [inline index.html:34]: - [ ] Kindly add image as a background image with CSS and not as img, please do this as this is part of the project requirement

File: index.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+              <li class="developer-experience-list-item">HTML</li>
+              <li class="developer-experience-list-item">CSS</li>
+            </ul>
+          </div>
+          <div class="developer-experience-container">
+            <img
+              class="developer-experience-shadow"
+              src="./images/header@2x.png"
+              alt=""
+            />
```
Comment (prolajumokeoni) [inline index.html:64]: - [ ] Kindly add image as a background image with CSS and not as img, please do this as this is part of the project requirement

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @learnwithalfred, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used 


## Required Changes ♻️
_Check the comments under the review._ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+      </header>
+      <section class="developer">
+        <div class="developer-info">
+          <img
+            class="developer-info-shadow"
+            src="./images/header@2x.png"
+            alt=""
+          />
+          <p class="developer-info-name">Laura Jong</p>
+          <p class="developer-info-title">Software Developer</p>
```
Comment (prolajumokeoni) [inline index.html:36]: - [ ] Kindly make the main name a h1 tag and the subtitle any hx tag either h2-h6 tag

File: index.html (hunk 1 above)
Comment (learnwithalfred) [reply index.html:26]: Thanks
I have wrapped the elements with a nav tag

File: index.html (hunk 2 above)
Comment (learnwithalfred) [reply index.html]: It is done

**Review by ShahierNashaat — APPROVED** [review]
Hi @learnwithalfred , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**